woocommerce/woocommerce-android
# [REST API] Analytics for application passwords web authorization

Part of #8559, a continuation for #8618


### Description


This adds the analytics for events related to webview-based REST API login.

Conversation in p1679454856318259-slack-C046HDZL87J

### Testing instructions


First, please open #8618 and have a look at the three test cases.

1. For the **"Success path"** case, please make sure the following are tracked: `application_passwords_authorization_web_view_shown`, `application_passwords_authorization_approved`
2. For the **"Failure path 1 : Application passwords authentication rejected"** case, please make sure the following are tracked: `application_passwords_authorization_web_view_shown`, `application_passwords_authorization_rejected`
3. For the **Failure path 2: Application passwords disabled** case, please make sure the following is tracked: `application_passwords_authorization_url_not_available`

### Images/gif
